clients/drcachesim/simulator/tlb_simulator.cpp

```diff
@@ -132,6 +154,23 @@ tlb_simulator_t::~tlb_simulator_t()
     delete[] lltlbs_;
 }
 
+std::string
+tlb_simulator_t::create_v2p_from_file(std::ifstream &v2p_file)
+{
+    // If we are not using physical addresses, we don't need a virtual to physical mapping
+    // at all.
+    if (!knobs_.use_physical)
+        return "";
+
+    std::string error_str = simulator_t::create_v2p_from_file(v2p_file);
+    if (!error_str.empty()) {
+        return error_str;
+    }
+    // Overwrite tlb_simulator_t page size with simulator_t page size.
+    knobs_.page_size = page_size_;
```

Comment:
I believe we should overwrite the `tlb_simulator_t.page_size` when using a given v2p.textproto mapping.
Using a different page size would require a new mapping right?

Reply:
Done.